README.md

```diff
@@ -1,62 +1,65 @@
-at their own discretion.
+Each project and working group is autonomous. The scope of work and 
+autonomy is described in their charter. The TSC works to increase communication 
+and collaboration between all these parts.
 
-## Top-Level WGs and TLPs
+Adding a new project or working group can be done through a pull request. Since
+each group is autonomous any new group taking on responsibility from another
+group would need that group's approval.
 
-* [Working Groups](WORKING_GROUPS.md)
-* Mentors
+## Projects
 
-    Project mentorship is not a technical role. In fact, mentors are
-    discouraged from giving technical advice to projects. Instead, the
-    purpose of mentorship is to encourage and improve a projects ability
-    to be participatory, transparent, and effective. Mentors are there to
-    help projects adopt and iterate on policies and processes that achieve
-    these goals and eventually allow them to graduate the incubation phase.
+* Core
+* Website
+* Streams
+* Docker
+* nan
 
-      * Mikeal Rogers (@mikeal)
-* Top-Level Projects
- * Core TLP
-  * Core WGs (streams, http, Intl)
+## Working Groups
 
-## Policy Change Proposal Process
+* Inclusivity
+* Build
+* Tracing
+* i18n
+* Evangelism
+* Roadmap
+* Addon API
+* Benchmarking
+* Post-mortem
+* Intl
+* HTTP
+* Documentation
+* Promises
+* Security
 
-The Node.js TSC is chartered to oversee the technical governance of all Top
-Level Projects and Working Groups under the Node.js Foundation. The TSC
-establishes the default governance, conduct, and licensing policies for all Top
-Level Projects. Top Level Projects and Working Groups have broad powers of
-self-governance.
+## Mentorship Program
 
-To propose a change or addition to policies or processes that are intended to
-cover all Top Level Projects and Working Groups in the foundation, a PR should
-be opened in the `nodejs/TSC` repository.
+The TSC administers a mentorship program for critical projects in the ecosystem.
+Projects are assigned mentors from the TSC to adapt modern best practices to the 
+specific needs of each project.
 
-The pull request can be labeled `tsc-agenda` to request that it be put on the
-agenda for the next TSC meeting.
+* libuv
+* Express
 
-The Node.js Foundation Board of Directors retains certain rights (especially
-legal considerations). If the TSC endorses a proposal, they will escalate to the
-Node.js Foundation Board of Directors when required to do so.
+## TSC Members
 
-In some cases, existing individual groups have the right to refuse changes to
-their charters. The TSC can not mandate existing working groups alter their
-charters. If such a situation arises, the TSC may decide to revoke the group's
-charter.
+TSC members are responsible for administration and communitication. The role is 
+mostly administrative and is responsible for admitting new projects, working groups 
+and representing them to  Node.js Foundation Board of Directors.
```

Comment:
Would/should a current list go here? This is what I would expect/hope to find here.